docs/data/joy/main-features/color-inversion/ColorInversionNavigation.js

```diff
@@ -131,28 +137,27 @@ export default function ColorInversionNavigation() {
         </Card>
       </Sheet>
       <Sheet
-        variant="soft"
-        color="neutral"
+        variant="solid"
         invertedColors
-        sx={(theme) => ({
+        sx={{
           p: 2,
           display: 'flex',
           flexDirection: 'column',
+          alignItems: 'center',
           gap: 2,
-          ...(color !== 'neutral' && {
-            bgcolor: `${color}.900`,
-          }),
+          bgcolor: `${color}.900`,
+
+          '& .MuiBadge-root': { '--Badge-ringColor': 'unset' },
```

Comment:
```suggestion
          '& .MuiBadge-root': { '--Badge-ringColor': '#FFF' },
```

Having the ring color in white will be mostly beneficial for the active green badge.